RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -29,4 +29,4 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603])
+                                                       872439603,872422436])
```

Comment:
the same comment as in https://github.com/cms-sw/cmssw/pull/25186/files#r232349972 applies here

Reply:
Until plans for a new MiniAOD in 102X and a new era is defined, this backport may stay on the todo list (hold?) then.